airflow/www/static/js/gantt.js

```diff
@@ -204,7 +204,7 @@ d3.gantt = () => {
       .on('mouseover', tip.show)
       .on('mouseout', tip.hide)
       .on('click', (d) => {
-        callModal(d.task_id, d.execution_date, d.extraLinks, undefined, undefined, d.run_id);
+        callModal(d.task_id, d.execution_date, d.extraLinks, undefined, undefined, d.run_id, d.map_index);
```

Comment:
```suggestion
        // eslint-disable-next-line max-len
        callModal(d.task_id, d.execution_date, d.extraLinks, undefined, undefined, d.run_id, d.map_index);
```

linter is angry about this. Let's ignore it for now. In a later PR, I would like to use object props here to avoid passing `undefined` and make it clearer which arg is what.